src/TRestGeant4ParticleCollection.cxx

```diff
@@ -39,15 +36,15 @@ TRestGeant4ParticleCollection* TRestGeant4ParticleCollection::instantiate(std::s
         // naming convension: TRestGeant4ParticleCollectionXXX
         // currently supported generator: decay0
         // in future we may add wrapper of generators: cry(for muon), pythia(for HEP), etc.
-        model[0] = *REST_StringHelper::ToUpper(std::string(&model[0], 1)).c_str();
+        model[0] = *REST_StringHelper::ToUpper(string(&model[0], 1)).c_str();
         TClass* c = TClass::GetClass(("TRestGeant4ParticleCollection" + model).c_str());
-        if (c != NULL)  // this means we have the package installed
+        if (!c)  // this means we have the package installed
```

Comment:
This is correct?
 if (c != NULL)  should translate to if(c)

Reply:
True, there should be a way to do this simple changes from the GUI...

Reply:
Do you want me to give a try? I never did in github but cannot be much different than gitlab

Reply:
Done, please check

Reply:
Did you do it from this page? or did you have to go to the repo, select the branch, find the file and change it?

Reply:
I did from the web page